editor/configs/emsColumns.tsx

```diff
@@ -64,6 +64,11 @@ export const ALL_EMS_COLUMNS = {
     path: "id",
     label: "ID",
     sortable: true,
+  },
+  id_hyperlinked: {
+    path: "id",
+    label: "ID",
+    sortable: true,
```

Comment:
curious how y'all feel about repeating columns when they have different formatting/definition needs. this is the first time we're doing things this way. for crashes, we have a totally separate column config for the crashes list and crash details, with a bit of redundancy between them